### PR TITLE
trim project_ids

### LINKED ISSUE #39: Trim project IDs when extracting from profile configuration
Hello,

we ran into an issue when we added a project id to an existing profile and separated it with a comma and a space. The profile was not used for processing the donation.

I think we should add a little fix for that, because for most people it feels common to use a space after a comma.

### PR DESCRIPTION
**Pull request by MarcMichalsky**
fix #39 

I tested it on our development server. It works fine.

### CODE REVIEW

File: CRM/Twingle/Profile.php [hunk 1]
```diff
@@ -66,6 +66,9 @@ public function __construct($name, $data) {
   public function matches($project_id) {
     $selector = $this->getAttribute('selector');
     $project_ids = explode(',', $selector);
+    for ($i=0; $i<sizeof($project_ids); $i++) {
+      $project_ids[$i] = trim($project_ids[$i]);
+    }
```
Comment (jensschuppe) [inline CRM/Twingle/Profile.php:71]: How about using an array utility function instead?

```php
$project_ids = array_map(
  function($project_id) {
    return trim($project_id);
  },
  explode(',', $selector)
);
```

Comment (MarcMichalsky) [reply CRM/Twingle/Profile.php:71]: Yes! This is definitely much more elegant!

Comment (MarcMichalsky) [reply CRM/Twingle/Profile.php:71]: Okay, I tested it. This solution works as well.

**Comment by jensschuppe** [comment]
Thanks @MarcMichalsky!

**Pull request merged**